oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -153,6 +163,85 @@ public static ImpersonatedCredentials create(GoogleCredentials sourceCredentials
         .build();
   }
 
+  /**
+   * Returns the email field of the serviceAccount that is being impersonated.
+   *
+   * @return email address of the impesonated service account.
+   */
+  @Override
+  public String getAccount() {
+      return this.targetPrincipal;
+  }
+
+  /**
+   * Signs the provided bytes using the private key associated with the impersonated
+   * service account
+   *
+   * @param toSign bytes to sign
+   * @return signed bytes
+   * @throws SigningException if the attempt to sign the provided bytes failed
+   * @see <a href="https://cloud.google.com/iam/credentials/reference/rest/v1/projects.serviceAccounts/signBlob>Blob Signing</a>
```

Comment:
Oops, one more nit, otherwise LGTM.
```suggestion
   * @see <a href="https://cloud.google.com/iam/credentials/reference/rest/v1/projects.serviceAccounts/signBlob">Blob Signing</a>
```